qt/FilterView.cc

```diff
@@ -0,0 +1,207 @@
+// This file Copyright © 2012-2022 Mnemosyne LLC.
```

Comment:
Unless you have somehow written this code in 2012 and transferred rights backs then, this needs to change.